python/requirements.txt

```diff
@@ -1,5 +1,5 @@
 Cython>=0.22
-pystan>=2.14
+cmdstanpy==0.4
```

Comment:
Not sure we can completely rip out pystan just yet. I see we need to figure out how to make both backends optional so that pystan is optional, though. I haven't seen a Python package with an either/or optional dependency before.

Reply:
@seanjtaylor  how about we parametrized install command like this:
`pip install fbprophet --install-option='--with-cmdstanpy'` 
(it seems doable: https://pip.pypa.io/en/stable/reference/pip_install/#per-requirement-overrides)
And then based on the parameter we would create either pystan or cmdstan model.

Reply:
That looks great to me!